01-messaging.md

```diff
@@ -224,12 +224,12 @@ Various fundamental types are referred to in the message specifications:
 * `u32`: a 4 byte unsigned integer
 * `u64`: an 8 byte unsigned integer
 
-Inside TLV records which contain a single value, leading zeros in
-integers can be omitted:
+For the final value in TLV records, truncated integers may be used. leading zeros in
```

Comment:
nit:
```suggestion
For the final value in TLV records, truncated integers may be used. Leading zeros in
```

Reply:
Damn it ^^! I'll forcefully amend that to avoid a useless commit.

Reply:
Don't hesitate to just squash everything to a single commit :)